src/pages/ShoppingCart.page.js

```diff
@@ -0,0 +1,42 @@
+import { InventoryPage } from './Inventory.page';
+
+export class ShoppingCartPage extends InventoryPage {
+    url = '/cart.html';
+
+    cartItemSelector = '.cart_item';
+
+    removeItemSelector = '[id^="remove"]';
+
+    get headerTitle() { return this.page.locator('.title'); }
+
+    get cartItems() { return this.page.locator('.cart_item'); }
```

Comment:
you can use `cartItemSelector` here

Reply:
Hey @ovartem !
don't clearly understand, why better to set selector with using cartItemSelector here

Reply:
get cartItems() { return this.page.locator(this.cartItemSelector); }